sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
     <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'net47'"/>
+     <ProjectReference Include="../../../resources/Azure.ResourceManager.Resources/src/Azure.ResourceManager.Resources.csproj" />
```

Comment:
All track 2 data-plane test projects use this project. Perhaps only include this in test projects for mgmt-plane? This will complicate the dependency graph MSBuild will have to calculate and slow builds. Perhaps go the other way: have this project reference automatically include Azure.Core.TestFramework to limit scope.

Reply:
I'm thinking the best way to do this may be as was suggested by either @weshaggard or @pakrym , forgot who. That is, to make a small rest client to do this for us, since we do not want cross dependencies and the unit tests that run in Azure.ResourceManager.Resources can get out of a sync with a released nuget package, which will cause these test fail making it fragile to import a nuget package here (since this test framework is depended a upon by Azure.ResourceManager.Resources).